src/client.js

```diff
@@ -247,38 +265,43 @@ _.extend(bulkXHR.prototype, {
 });
 
 /**
- * SugarCRM Javascript API allows users to interact with SugarCRM instance via its REST interface.
+
+ *
+ *
+ */
+
+/**
+ * SugarCRM Javascript API allows users to interact with SugarCRM instance
+ * via its REST interface.
  *
- * Use {@link Api#getInstance} method to create instances of Sugar API.
- * This method accepts arguments object with the following properties:
- * <pre>
+ * Most of Sugar API methods accept `callbacks` object:
+ * ```
  * {
- *   serverUrl: Sugar REST URL end-point
- *   platform: platform name ("portal", "mobile", etc.)
- *   keyValueStore: reference to key/value store provider used to read/save auth token from/to
- *   timeout: request timeout in seconds
+ *     success: function(data) {},
+ *     error: function(error) {},
+ *     complete: function() {},
```

Comment:
Do you think it's worthwhile using [the typedef tag](usejsdoc.org/tags-typedef.html) to make this into a custom type and then referring to it whenever necessary? There's arguments both for and against.